javascript/javascript3/week1/Weather_app/geoloc.js

```diff
@@ -0,0 +1,29 @@
+// Working further -> On Map and Local Storage
+// Currently could get current location latitude and longitude and fetch data based on it
+
+const BASE_URL = "https://api.openweathermap.org/data/2.5/weather";
+
+function getGeo() {
+    navigator.geolocation.getCurrentPosition(getCurrentLoc)
```

Comment:
Does this need to be a function?

Reply:
I am not sure... I was stuck on this one. I saw this in one of the tutorials on google and it worked. Should I try it in some other way?

Reply:
No, it's fine. I was just saying instead of making it a function and calling it you could just run `navigator.geolocation.getCurrentPosition(getCurrentLoc)` where you call it.